playerSDK/src/main/java/com/kaltura/playersdk/players/KCCRemotePlayer.java

```diff
@@ -261,6 +260,11 @@ public boolean isPlaying() {
         return false;
     }
 
+    @Override
+    public void switchToLive() {
+
```

Comment:
Not sure this should really be empty. Chromecast can switch from timeshift to live, right?